lib/src/arg_parser.dart

```diff
@@ -327,4 +352,8 @@ class ArgParser {
     }
     return null;
   }
+
+  /// Finds the option whose name or alias matches [name], or `null` if no
+  /// option has that name or alias.
+  Option? findByNameOrAlias(String name) => options[_aliases[name] ?? name];
```

Comment:
Does this need to be public?

Reply:
That is the simplest way, yes. We do also already have a public `findByAbbreviation` method.

We need to do similar logic inside the `parser.dart` library, which would otherwise be duplicated.